MUXControlsInnerLoop.sln

```diff
@@ -342,6 +342,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TabView", "TabView", "{B3E6
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "TabView", "dev\TabView\TabView.vcxitems", "{B9F81FEF-1E8D-4FE1-A46B-7002D4C109D2}"
 EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "TabView_APITests", "dev\TabView\APITests\TabView_APITests.shproj", "{2F4E95E9-F729-481C-B9AA-C9BEC91AE395}"
+EndProject
```

Comment:
Aren't those changes that I would need to revert anytime I am using the innerloop without TabView?

Reply:
This in a permanent change which adds the TabView's API test project to the innerloop solution (missing yet in VS).

On that note, it might be better to make this a separate commit to the master branch depending on how long it will take to merge this PR into master. Otherwise, we could end up seeing multiple TabView PRs which modify the API test project and each wouldthen have the to make the same changes -- if the contributor uses the innerloop.

Reply:
I think that that change makes more sense to have given that this PR might take some time and contributors would be left without those changes until then. In addition to that, it's also only related to the bug you are fixing in the sense that it is the same control, not much more.

Reply:
We already have at least *two* outstanding TabView PRs which add the missing API test project to the innerloop solution so the situation is already less than ideal. I will create a standalone PR for this in the next few minutes.

Reply:
Standalone PR https://github.com/microsoft/microsoft-ui-xaml/pull/3770 has been created.